graph/api/src/main/java/org/jboss/windup/graph/GraphUtil.java

```diff
@@ -29,49 +29,49 @@ private static final void vertexAsString(Vertex vertex, int depth, String withEd
         String indent = StringUtils.repeat("    ", atLevel);
         if (vertex == null)
         {
-            sb.append("\n").append(indent).append("(vertex == null)");
+            sb.append(System.lineSeparator()).append(indent).append("(vertex == null)");
             return;
         }
         if (visitedIDs.contains(vertex.getId()))
         {
-            sb.append("\n").append(indent).append("" + vertex.getId());
+            sb.append(System.lineSeparator()).append(indent).append("" + vertex.getId());
             return;
         }
 
         visitedIDs.add(vertex.getId());
 
-        sb.append("\n").append(indent).append("v #").append("" + vertex.getId()).append(" {");
+        sb.append(System.lineSeparator()).append(indent).append("v #").append("" + vertex.getId()).append(" {");
         boolean hasProps = !vertex.getPropertyKeys().isEmpty();
         boolean hasEdges = vertex.getEdges(Direction.IN).iterator().hasNext()
                     || vertex.getEdges(Direction.OUT).iterator().hasNext();
 
         for (String propKey : vertex.getPropertyKeys())
         {
-            sb.append("\n    ").append(indent).append(propKey).append(": ").append("" + vertex.getProperty(propKey));
+            sb.append(System.lineSeparator()).append(indent).append(propKey).append(": ").append("" + vertex.getProperty(propKey));
         }
 
         if (withEdgesOfLabel == null || depth == 0)
         {
             if (hasProps)
-                sb.append("\n    ").append(indent);
+                sb.append(System.lineSeparator()).append(indent);
             if (hasEdges)
```

Comment:
You have omitted the spaces, making the log less radable.

Reply:
new line with tab the result will look very the same IMO